nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -500,9 +526,13 @@ buildSubGraphs(const TfOpNodes &nodes, const TfOpIdxMap &map,
 
   /// @todo extract this to buildSubgraph if there is one or more subgraph
   auto name = fbb.CreateString("main");
+  auto inputs = fbb.CreateVector(map.getInputs());
+  auto outputs = fbb.CreateVector(map.getOutputs());
 
   auto builder = tflite::SubGraphBuilder(fbb);
   builder.add_tensors(tensors);
+  builder.add_inputs(inputs);
+  builder.add_outputs(outputs);
```

Comment:
does Tflite need this inputs and outputs of model?

Reply:
Yes, they are identified specially.